README.org

```diff
@@ -2,9 +2,14 @@
 
 * What is it
 
-  This repository is software for the Raspberry Pi board of
-  SC-Sat1. SC-Sat1 is a 3U CubeSat that Space Cubics launches to
-  demonstrate SC-OBC in space.
+  This repository hosts software for the Raspberry Pi board of SC-Sat1.
+  As you can see from the picture, it has a Raspberry Pi Zero W and a
```

Comment:
We use a `Raspberry Pi Zero 2 W` not `Raspberry Pi Zero W`.

Reply:
oops. fixed.